clients/src/main/java/org/apache/kafka/clients/consumer/WakeupException.java

```diff
@@ -14,7 +14,7 @@
 
 import org.apache.kafka.common.KafkaException;
 
-public class ConsumerWakeupException extends KafkaException {
+public class WakeupException extends KafkaException {
```

Comment:
Ah, right, I was just trying to minimize `consumer` related imports. We need to catch the exception in Copycat code, so we end up with `consumer` imports there. For most classes it doesn't matter where exactly they are located for now since they are internal anyway, but this one is external interface and would be more of a pain to move later (although it also isn't used publicly in any way in Copycat).